src/components/dialogs/DialogLinkReliability.vue

```diff
@@ -306,10 +312,9 @@ export default {
 				},
 			)
 
-			this.running = false
-
 			if (response.success) {
 				this.statistics = response.result
+				this.running = true
```

Comment:
...and this doesn't make sense. Awaiting `checkLinkReliability` is the entire process, so after it returns, the check is done and is no longer running.

Reply:
Oh didn't know that I thought the api was only used to start the process!